policy/resolver_test.go

```diff
@@ -264,6 +264,79 @@ policies:
 	}
 }
 
+func TestResolve_IgnoredQuery(t *testing.T) {
+	b := parseBundle(t, `
+owner_mrn: //test.sth
+policies:
+- uid: policy-1
+  owner_mrn: //test.sth
+  groups:
+  - type: chapter
+    filters: "true"
+    checks:
+    - uid: check1
+      mql: 1 == 1
+`)
+
+	srv := initResolver(t, []*testAsset{
+		{asset: "asset1", policies: []string{policyMrn("policy-1")}},
+	}, []*policy.Bundle{b})
+
+	bundleMap, err := b.Compile(context.Background(), schema, nil)
+	require.NoError(t, err)
+
+	// we get the asset policy, manually apply the ignored action and store it back
+	ap, err := srv.DataLake.GetRawPolicy(context.Background(), "asset1")
+	require.NoError(t, err)
+	require.NotNil(t, ap)
+	ap.Groups[0].Checks = []*explorer.Mquery{
```

Comment:
does setting action in the policy directly not work like we do in `TestResolve_DisabledQuery`?

Reply:
yep, had. to play a bit with it but got it working